File: java/org.chabu.test.node/src/org/chabu/nwtest/Parameter.java
package org.chabu.nwtest;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlTransient;

public abstract class Parameter {
	
	@XmlElement(name="Name")
	private String name = "";
	
	public Parameter() {
	}

	public Parameter(String name) {
		this.name = name;
	}
	
	@XmlTransient
	public String getName() {
		return name;
	}
}

File: c-sharp/org.chabu.test.director/prot/Parameter.cs
using System;
using System.Xml.Serialization;

namespace org.chabu.test.director.prot
{
    [Serializable]
    [XmlInclude(typeof(ParameterValue))]
    [XmlInclude(typeof(ParameterWithChilds))]
    public abstract class Parameter
    {
        public string Name { get; set; }
    }
}